app/experimenter/nimbus-ui/src/hooks/useCommonForm/useCommonFormMethods.tsx

```diff
@@ -0,0 +1,132 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This seems pretty nicely separated out from useCommonForm too, basically what I think I handwaved at without helpfully explaining in much detail in Slack 😅

Reply:
Hahaha, you know what's funny is I went and reread what you told me a few hours ago after I finished this up and was like hey, I ended up doing exactly what Les vaguely said! 😆